pkg/netset/transportset.go

```diff
@@ -55,6 +82,10 @@ func AllTransportSet() *TransportSet {
 	return AllOrNothingTransport(true, true)
 }
 
+func EmptyTransportSet() *TransportSet {
```

Comment:
Suggest renaming to `NoTransports`.
Also, rename `AllTransportSet()` to `AllTransports()`

Reply:
fixed